static/locales/en-US.yaml

```diff
@@ -481,13 +482,14 @@ Settings:
         Name: None
   Privacy Settings:
     Privacy Settings: Privacy
-    Remember History: Remember History
+    Remember History: Remember Watch History
+    Remember Search History: Remember Search History
     Save Watched Progress: Save Watched Progress
     Save Watched Videos With Last Viewed Playlist: Save Watched Videos With Last Viewed Playlist
-    Clear Search Cache: Clear Search Cache
-    Are you sure you want to clear out your search cache?: Are you sure you want to
-      clear out your search cache?
-    Search cache has been cleared: Search cache has been cleared
+    Clear Search History and Cache: Clear Search History and Cache
```

Comment:
Did we want to remove the old key for the other locales? Or is the plan to keep it for now so it's easier to update the translation?

Reply:
Yes, I was thinking the latter.

Reply:
Sounds good, approving now